### ISSUE TITLE
Unknown prop `onRemove` on <div> tag. Remove this prop from the element. 

### ISSUE THREAD

**Issue by srakshit** [joshcarr/react-structured-filter#8]
Hi @joshcarr 

I'm getting the following error - 

![image](https://cloud.githubusercontent.com/assets/2448505/17114264/076c5150-52a6-11e6-8179-b47b44bd04b1.png)

Seems like its coming out of the render function of token.js

``` javascript
 render() {
    const { category, operator, value } = this.props.children;
    return (
      <div {...this.props} className="typeahead-token">
        <span className="token-category">{ category }</span>
        <span className="token-operator">{ operator }</span>
        <span className="token-value">{ value }</span>
        { this._makeCloseButton() }
      </div>
    );
  }
```

Could you please help.

**Comment by EXPEvgowdaks** [comment]
Even am facing the same issue. It started after upgrading react to 15.2.X.

**Comment by granda** [comment]
Still present in `react@15.3.0` 😢 

Edit: Attempted with example present in [`docs@c91aa9b`](https://github.com/joshcarr/react-structured-filter/blob/c91aa9b8e05074b87209cb434c0278a7a20f3a86/docs/index.md). Also, I'd like to thank you personally for the work you put into supplying documentation and uploading this to npm. 🙌🏼

**Comment by joshcarr** [comment]
Fixed with a new version on npm v4.3.1 efb8c77

**Comment by srakshit** [comment]
Hi @joshcarr 

Thanks for fixing in the _src/tokenizer/token.js_. 

But as in your _package.json_ `main` is referring to `lib/main.js` it still picks up the old code as the lib directory still has the props in div. Could you please build the lib directory.

**Comment by joshcarr** [comment]
Thanks for the catch, fixed in 25a4d6014d75f5f2fd7d704b98673fef85bf5e2c and published to npm as v4.3.2
